File: dotnet/DesignPatterns/StructuralDesignPatterns/FacadeDesignPattern/FacadeDesignPattern.Example1/Models/Amplifier.cs
namespace FacadeDesignPattern.Example1.Models;

public class Amplifier
{
	public void On() => Console.WriteLine("Amplifier is turned on.");
	public void Off() => Console.WriteLine("Amplifier is turned off.");
	public void SetVolume(int level) => Console.WriteLine($"Amplifier volume is set to {level}.");
}

File: java/DesignPatterns/StructuralDesignPatterns/FacadeDesignPattern/FacadeDesignPattern.Example1/src/main/java/structural/facade/example1/Amplifier.java
package structural.facade.example1;

public class Amplifier {
    public void on() { System.out.println("Amplifier is turned on."); }
    public void off() { System.out.println("Amplifier is turned off."); }
    public void setVolume(int level) { System.out.println("Amplifier volume is set to " + level + "."); }
}
